redash/settings/__init__.py

```diff
@@ -2,19 +2,7 @@
 from funcy import distinct, remove
 
 from .helpers import fix_assets_path, array_from_string, parse_boolean, int_or_none, set_from_string
-from .organization import DATE_FORMAT
-
-
-def all_settings():
-    from types import ModuleType
-
-    settings = {}
-    for name, item in globals().iteritems():
-        if not callable(item) and not name.startswith("__") and not isinstance(item, ModuleType):
-            settings[name] = item
-
-    return settings
-
+from .organization import DATE_FORMAT  # noqa
```

Comment:
?

Reply:
The `# noqa` comment prevents flake8 to mark this as an error since the value isn't used.  My editor is using flake8 for inline checks, do you want me to revert it?

Reply:
I was asking about the import itself, but after the fact noticed that it was there already just moved...